README.md

```diff
@@ -56,17 +56,12 @@ with:
     evmPrivateEncrypted: "encrypted-key"
     erc20RewardToken: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
     incentives:
-      enabled: true
       requirePriceLabel: true
       contentEvaluator:
-        enabled: true
       userExtractor:
-        enabled: true
         redeemTask: true
       dataPurge:
```

Comment:
Can you explain what these do?

Reply:
They enable the modules inside this plugin. They allow for specific configuration for each step.

Reply:
Content evaluator: evaluates and rates comments (scores).
Redeem task: Is the task redeemable, e.g. can the user collect the bounty?
dataPurge: Removes the data in the comments that we do not want to be processed.

There are comments inside each of these in the code.